libs/configuration/configuration-endpoints/src/main/kotlin/net/corda/libs/configuration/endpoints/v1/ConfigRPCOps.kt

```diff
@@ -48,7 +48,7 @@ interface ConfigRPCOps : RpcOps {
                 match the version stored in the database for the corresponding section or -1 if this is a new section 
                 for which no configuration has yet been stored.""")
         request: UpdateConfigParameters
-    ): UpdateConfigResponse
+    ): ResponseEntity<Any>
```

Comment:
Why not to keep it as:
```suggestion
    ): ResponseEntity<UpdateConfigResponse>
```

appreciate in some cases no result will be returned post HTP invocation due to HTTP server restart. But there will be also config sections which HTTP Server does not care about and response will be sent back to client as normal.

Also type `Any` is not good for OpenAPI generation.

Reply:
We;'re just hacking some solutions to a stability issue. Hold off any official reviews for now.